civiawards.php

```diff
@@ -171,6 +169,14 @@ function civiawards_civicrm_permission(&$permissions) {
     'CiviAwards: Create/Edit awards',
     ts('Allows a user to create or edit awards'),
   ];
+
+  $permissions['access review custom field set'] = [
+    'CiviAwards: Access review fields ',
```

Comment:
Same as above, don't we need localisation tag for permission name?

Reply:
I'll add for this.